OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -847,6 +838,24 @@ public void ResolveOrder(Actor self, Order order)
 			}
 		}
 
+		void DoMove(Target target, bool queued)
```

Comment:
Naming nit: aim for something like `void MoveInner(Actor self, Target target, bool queued)`

Reply:
This is no longer relevant in the current version.